packages/api-db-mongodb/src/migration.ts

```diff
@@ -613,6 +614,33 @@ export const Migrations: Migration[] = [
         {$set: {oauth2Accounts: []}}
       )
     }
+  },
+  {
+    //  Set paymentProviderCustomers to an empty array
+    version: 15,
+    async migrate(db, locale) {
+      const users = await db.collection(CollectionName.Users)
+
+      const allUsers: DBUser[] = await users.find({}).toArray()
+
+      for (const user of allUsers) {
+        const paymentProvidersCustomersArray: PaymentProviderCustomer[] = []
+        const paymentProviderCustomers = Object.keys(user.paymentProviderCustomers)
+        paymentProviderCustomers.forEach(ppc => {
+          // @ts-ignore
+          const temp = user.paymentProviderCustomers[ppc]
```

Comment:
the name of the `temp `variable is not very descriptive, could it be more self-explanatory or can you explain what it describes?